### ISSUE TITLE
Prioritizing publications

### ISSUE THREAD

**Issue by Rajat-Sen** [StatCan/StatcanPublicationUAT#145]
On the Publication screen, when the user is navigating through publications the voiceover skips the publications focuses on slider and the Tab Bar. 

The intended flow of navigation while using voiceover should be from navigating from one publication to another publication even if they are under the publication screen. It should not focus on slider and should stay focused on publications. Additionally, only focus on Tab bar, when the user is sliding his finger over the screen.

**Comment by ForumG-SC** [comment]
@Rajat-Sen Not sure if this was tested properly. I don't see any issue with the navigation on the publication screen. When testing VoiceOver navigation is not automatic, you have to manually select wherever you want to navigate. See the attached screen recording:
[RPReplay-Final1640860210.mov.zip](https://github.com/StatCan/StatcanPublicationUAT/files/7792529/RPReplay-Final1640860210.mov.zip)

**Comment by Rajat-Sen** [comment]
@ForumG-SC, I do understand the point where you are coming from.

Since, the testing is done on iPhone I would like you try out Apple news, Google news, CBC news. In any of this app if you turn on Voiceover on, with only your thumb swipe from left to right and keep swiping. You will see the focus going from one card to another, the scrolling is done automatically and the focus does not go to the slider bar at all.

Now, I understand the user can swipe with more than one finger. But, that behaviour is for web browsers. Thanks!

**Comment by ForumG-SC** [comment]
@Rajat-Sen I tested Apple News app like you mentioned and I do see that the focus does goes on tab bar and slide bar, please see the screen recording of it.

https://user-images.githubusercontent.com/84411852/147839101-7c84dce4-85ac-48bd-b713-181d168c1a6f.mov

**Comment by ForumG-SC** [comment]
Closing this issue as the focus issue has been resolved.
